src/tracing/mod.rs

```diff
@@ -173,6 +173,19 @@ impl TracingInspector {
         }
     }
 
+    /// Manually the gas limit of the debug root trace.
```

Comment:
```suggestion
    /// Manually set the gas limit of the debug root trace.
```

Reply:
Hi @mattsse, thanks so much for your quick comment. I have done some easy cleanup. Since I'm just getting started with Rust and Web3, please let me know somewhere can be improved or fixed. 😄

Reply:
Hi @mattsse, I think it is ready now.